datafusion/expr-common/src/signature.rs

```diff
@@ -378,6 +379,9 @@ impl TypeSignatureClass {
             TypeSignatureClass::Binary => {
                 vec![DataType::Binary]
             }
+            TypeSignatureClass::Decimal => vec![Decimal128Type::DEFAULT_TYPE],
+            TypeSignatureClass::Float => vec![DataType::Float64],
+            TypeSignatureClass::Numeric => vec![DataType::Float64],
```

Comment:
```suggestion
            TypeSignatureClass::Numeric => vec![DataType::Int64, DataType::Float64],
```
to show the difference between TypeSignatureClass::Float and TypeSignatureClass::Numeric